terminology/README.md

```diff
@@ -0,0 +1,15 @@
+# Terminology
+
+The language we use has power. Being as concise and descriptive as possible helps to lessen ambiguity. This leads to shared understanding and better outcomes.
+
+## Terms
+
+* Use `blocklist`, and not `blacklist`.
+* Use `allowlist`, and not `whitelist`.
+* Avoid using `master` and `slave`. 
+	* For GitHub, use a term such as `main`, or `publish`, `production`, or `trunk`.
```

Comment:
Can we pick one? I've been using `main` in my personal projects, and much of the discussion online has been around `main` -- can we agree on that one?

Reply:
Totally. `main` works for me!